google-cloud-pubsub/support/doctest_helper.rb

```diff
@@ -406,6 +406,20 @@ def mock_pubsub
     end
   end
 
+  doctest.before "Google::Cloud::PubSub::Subscription#reload!" do
+    mock_pubsub do |mock_publisher, mock_subscriber|
+      mock_subscriber.expect :get_subscription, subscription_resp, [String, Hash]
+      mock_subscriber.expect :get_subscription, subscription_resp, [String, Hash]
+    end
+  end
+
+  doctest.before "Google::Cloud::PubSub::Subscription#refresh!" do
```

Comment:
We usually skip aliases. (Not a big deal, I will approve anyway in case you are in a hurry.)

Reply:
Skipped.